crates/sui/tests/snapshots/shell_tests__explicit_implicits__modern_cannot_use_legacy_name_as_replacement.sh.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: crates/sui/tests/shell_tests.rs
+description: tests/shell_tests/explicit_implicits/modern_cannot_use_legacy_name_as_replacement.sh
+---
+----- script -----
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# tests that building a package that has explicit deps (through dep-replacements) on legacy system names errors
+sui move --client.config $CONFIG build -p modern_using_legacy_name_as_replacement
+
+----- results -----
+success: false
+exit_code: 1
+----- stdout -----
+Error while loading dependency .: Dependency `MoveStdlib` is a legacy system name and cannot be used. See: https://docs.sui.io/guides/developer/sui-101/move-package-management#system-dependencies
```

Comment:
The "while loading dependency ." is weird - I'm surprised it's getting wrapped this way